app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/PrimaryContactDetails.scala

```diff
@@ -26,8 +26,11 @@ case class PrimaryContactDetails(
   email: Option[String] = None,
   phoneNumber: Option[String] = None,
   useRegisteredAddress: Option[Boolean] = None,
+  address: Option[Address] = None,
+  // The following fields are used for email verification only
   journeyId: Option[String] = None,
-  address: Option[Address] = None
+  prospectiveEmail: Option[String] =
+    None // Not mirrored in back end since only used during registration amendment
```

Comment:
This bit is potentially controversial. Are we happy to record the prospective email change here (alongside the journeyId of the email verification)? I would have quite liked to have refactored here in order to make it clearer that this stuff is temporary but I want to avoid another backwards-incompatible change to Registration.